edit-post/store/effects.js

```diff
@@ -29,7 +29,14 @@ const effects = {
 		}
 
 		// Allow toggling metaboxes panels
-		window.postboxes.add_postbox_toggles( select( 'core/editor' ).getCurrentPostType() );
+		// We need to wait for all scripts to load
+		// If the meta box loads the post script, it will already trigger this.
+		setTimeout( () => {
+			const postType = select( 'core/editor' ).getCurrentPostType();
+			if ( window.postboxes.page !== postType ) {
+				window.postboxes.add_postbox_toggles( postType );
+			}
+		} );
```

Comment:
Given how key this is to the core WordPress experience, is it possible to add a test for it?

Reply:
With a lot of mocks, I can probably add a unit test but I don't think it will be a valuable test because this test would test the implementation and not really the problem. 

I think an `e2e` test is better here but we have an issue :) We didn't do tests that install plugins yet I'm going to try.

Reply:
> We didn't do tests that install plugins yet I'm going to try.

I don't know that I'd install a third-party plugin (because that's likely to change over time). I'd probably create a small meta box PHP file and include it in the tests directory for occasional inclusion.

Reply:
I'm planning to use `EditFlow` for this test, Granted that it can change but I'm thinking not that often. If you'd like to help with adding a test plugin (also not certain how to enable this plugin in our current docker setup), I'd really appreciate it.

Reply:
I've created #5909. My comments shouldn't block merge